dmutils/s3.py

```diff
@@ -1,74 +1,87 @@
 from __future__ import absolute_import
 import os
-import boto
-import boto.exception
+import boto3
 import datetime
 import mimetypes
 import logging
 from dateutil.parser import parse as parse_time
+from six import text_type
 
-from boto.exception import S3ResponseError  # noqa
+# import this and catch it if you like but doing so you give up any pretense at boto version independence.
```

Comment:
`from botocore.exceptions import ClientError as S3ResponseError` is a bit of a weird lie though isn't it? It's the sort of thing that implies one way of working right up to the point that someone goes "oh it's a boto error, ok - `from boto.exception import S3ResponseError ... except S3ResponseError`, yeah I expect that works..." but I don't exactly have a better solution. I guess it's also the issue that it's not (necessarily) just `S3ResponseError` or `ClientError`, but other exceptions that leak out and apps might want to be able to catch...